### PR TITLE
165383342 Fix Login validations

### PR DESCRIPTION
**Pull request by davidokonji**
#### Type of change
- [x] Bug fix (none breaking change which fixes an issue)
#### How Has This Been Tested?
Screenshot 2019-04-16 at 16 03 56
Screenshot 2019-04-16 at 16 04 24

#### Checklist:
- [x] add fix to authentication validations
- [x] modify usages in every file needed

#### Story (PT)
[Delievers #165383342](https://www.pivotaltracker.com/story/show/165383342)

### CODE REVIEW

**Review by chukwudidu — APPROVED** [review]
@davidokonji Good job on this fix.

File: server/utils/users/user.util.js [hunk 1]
```diff
@@ -64,36 +49,47 @@ class userValidation {
...
-      lastname
-    } = req.body;
-    return isEmpty({
-      email,
-      password,
-      firstname,
-      lastname
-    }, res, next);
+  static async isSigninFieldEmpty(req, res, next) {
+    // const { email, password } = req.body;
```
Comment (samorahno) [inline server/utils/users/user.util.js]: kindly remove the comment on this line @davidokonji

**Review by samorahno — CHANGES_REQUESTED** [review]

File: server/test/clap.spec.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ describe('Articles routes', () => {
       .post('/api/v1/auth/login')
       .send({
         email: 'mofe@okoro.com',
-        password: 'soldier'
+        password: 'soldier123'
```
Comment (Chiazokam) [inline server/test/clap.spec.js:16]: Any particular reasons why you changed the passwords?

**Review by TheDrizzyWay — APPROVED** [review]
Good to see that you edited the password in every single test file affected by your change.

**Review by geekdanitek — APPROVED** [review]

File: server/utils/users/user.util.js (hunk 1 above)
Comment (davidokonji) [reply server/utils/users/user.util.js]: Thank you for the observation, would fixed that now

File: server/test/clap.spec.js (hunk 2 above)
Comment (davidokonji) [reply server/test/clap.spec.js:16]: @Chiazokam changed password because i added validation for alphanumeric values, so modified all seeded passwords

**Review by samorahno — APPROVED** [review]

File: server/test/clap.spec.js (hunk 2 above)
Comment (Chiazokam) [reply server/test/clap.spec.js:16]: Alright

**Review by Chiazokam — APPROVED** [review]

**Review by BukkyOmo — APPROVED** [review]

**Pull request merged**